### PR TITLE
 Turkish language added

### PR DESCRIPTION
**Pull request by GkhnGRBZ**
Description

Jellyseerr's Turkish translation has been completed via weblate and the necessary fields have been added to the "LanguageContext" and "_app.tsx" files.

To-Dos

- [ ] Successful build `pnpm build`
- [ ] Translation keys `pnpm i18n:extract`

### CODE REVIEW

File: src/i18n/locale/tr.json
Comment (fallenbagel) [inline src/i18n/locale/tr.json:1]: This should not be included if the weblate already has this

**Review by Fallenbagel — CHANGES_REQUESTED** [review]

File: src/i18n/locale/tr.json
Comment (G-grbz) [reply src/i18n/locale/tr.json:1]: I deleted the file but it still appears in the changes. Should I create a PR again?

Comment (gauthier-th) [reply src/i18n/locale/tr.json:1]: You should discard the change you made, from the develop branch, because this file already exists.

Comment (jellyseerr-weblate) [inline src/i18n/locale/tr.json:1]: Now the file is deleted. You should rebase this PR after discarding the changes

Comment (fallenbagel) [reply src/i18n/locale/tr.json:1]: Now the file is deleted. You should rebase this PR after discarding the changes

**Pull request closed without merging**